pkg/cmd/task/delete.go

```diff
@@ -125,6 +125,8 @@ func taskRunLister(cs *cli.Clients, ns string) func(string) ([]string, error) {
 		if err != nil {
 			return nil, err
 		}
+		//this is required as the same label is getting added for both task and ClusterTask
```

Comment:
Nit: include space after comment beginning
```suggestion
		// this is required as the same label is getting added for both task and ClusterTask
```

Nit: capitalize Task
```suggestion
		// this is required as the same label is getting added for both Task and ClusterTask
```